libraries/stdlib/src/kotlin/random/XorWowRandom.kt

```diff
@@ -6,11 +6,13 @@
 package kotlin.random
 
 /**
- * Random number generator, algorithm "xorwow" from p. 5 of Marsaglia, "Xorshift RNGs".
+ * Random number generator, using Marsaglia's "xorwow" algorithm
  *
- * Cycles after 2^160 * (2^32-1) repetitions.
+ * For more details, see Marsaglia, George (July 2003). "Xorshift RNGs". Journal of Statistical Software. 8 (14). doi:10.18637/jss.v008.i14
  *
- * See http://www.jstatsoft.org/v08/i14/paper for details.
+ * Available at https://www.jstatsoft.org/v08/i14/paper
+ *
+ * Cycles after 2^192 - 2^32 repetitions.
```

Comment:
Minor: this sentence would look better in the beginning of the details, i.e. right after the summary line.

Reply:
See if it's better now